src/wkt/src/rstruct.rs

```diff
@@ -110,14 +114,94 @@ impl crate::message::Message for ListValue {
     }
 }
 
+impl crate::message::Message for NullValue {
+    fn typename() -> &'static str {
+        "type.googleapis.com/google.protobuf.Value"
```

Comment:
I think I am missing the nuance here on why `ListValue` is `google.protobuf.ListValue` but things like `BoolValue` / `NullValue` are just `google.protobuf.Value`.

I am also not seeing why we need the special `NullValue` when we do not have a special `BoolValue`.

Reply:
So why isn't the typename s/Value/NullValue/ on this line?